core/input/generisdata_tech.prn

```diff
@@ -182,7 +184,7 @@ inco0               0.01       21600
 mix0                   0           0
 eta                 1.00        1.00
 omf                 0.84       0.025
-omv                    0           0
+omv                    0
```

Comment:
are you sure that here are all row correct? in .prn files we need to be careful with tabs

Reply:
hm, pretty sure. `Rscript start.R -q` works. No value is treated as 0. We decided to remove the 0 as it is not used for dac.